tests/conftest.py

```diff
@@ -70,8 +70,6 @@ def filter_access_token(interaction, current_cassette):
     ).split()
 }
 
-if placeholders["refresh_token"] != "placeholder_refresh_token":
```

Comment:
Why were these lines removed?

Reply:
When I added them a couple commits ago, the tests built fine. When I ran the tests today it would fail unless those lines were commented out.

Reply:
Why do the tests fail?

Reply:
Once I get access to a computer I will clone and test the commit to see why that doesn’t work

Reply:
Okay I know why now.

With this PR I wanted to make tests that would account for larger subreddits. To do that, it needed to run on my reddit account. The problem is I have 2FA on my reddit account and I'm not willing to remove it I moderate 200 subreddits 75 million combined subscribers) just to make the tests with password auth. So I added a way for the tests to accept a refresh token instead of username/password. I know I can do `password:code` but that only lasts for 10 minutes. Here is what it does with that code in there:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/15524072/71550019-dab9ee80-298c-11ea-9232-1b55bb83fd81.png">
Also, [this code here](https://github.com/LilSpazJoekp/praw/blob/Moderation-Streams/tests/integration/__init__.py#L36-L37) 

If you want to use refresh tokens you can't pass the password arg to the reddit class because this error will occur: `"{"error_description": "Only script apps may use password auth", "error": "unauthorized_client"}"`

So, to remedy this, I added in code to pop out the password arg if refresh_token is provided and I forgot to remove the code above.

Reply:
Actually, looking at it again, [this code](https://github.com/LilSpazJoekp/praw/blob/Moderation-Streams/tests/integration/__init__.py#L36-L37) does not do the same thing.

Reply:
Okay I dropped the code that adds support for using a refresh token instead of user/pass.

Reply:
We do not need to pass in usernames either, it just works with a refresh token, and the client_id/secret and user_agent.

Reply:
Yeah but I was having trouble getting that to work. If I pass in a token then the password can not be passed in later when using web apps, which is what I use because my apps need to be able to be authed with accounts other than mine.

Reply:
Also, I think adding the support for using a token in tests should be its own PR.

Reply:
> Also, I think adding the support for using a token in tests should be its own PR.

That is completely true and that should get merged beforehand.

Reply:
The problem is that it has been tricky for me to find a way to have both user/pass and tokens work concurrently. I regenerated the cassettes using user/pass and all test pass now. This pr is ready.

Reply:
@LilSpazJoekp I would think that all you have to do is check if a token is given, then you remove a password. However, what if they do not use a username and just do a refresh token like me? So I think we should check the username as well. However, it would help if I knew what the values were if there were placeholders.

Reply:
I tried that but if I make a cassette with a token then later on make a cassette with user/pass the token one will fail.

Reply:
So it's a cassette thing? ahh ok that's a problem.

Reply:
maybe a good idea would be to standardize everything using tokens. I'll make an issue and bring it up, but someone needs to re-run all the tests with tokens and then force the usage of tokens for future tests.

Reply:
Yes.